### PR TITLE
bug(search): remove native date field clear button

### LINKED ISSUE #309: Hide native browser clear button in "date" field
Currently if we have search form with input field of type "date", two clear buttons will appear (native, and dress-code), so we need to remove the native one

#### Current state:

pasted image 0

#### Suggested solution:
```css
:-webkit-clear-button {
  display: none;
}
```

### PR DESCRIPTION
**Pull request by LeoAref**
screen shot 2017-01-16 at 11 43 25

Closes #309

### CODE REVIEW

**Comment by LeoAref** [comment]
@gabrielhl @rbarilani @gargravarr

File: src/styles/molecules/_search-form.scss [hunk 1]
```diff
@@ -15,10 +15,15 @@
 }
 
 @mixin dc-search-form__input {
+    max-height: 36px;
     margin: 0;
     padding-right: 30px;
     -moz-appearance: none;
     -webkit-appearance: none;
+
+    &[type="date"]::-webkit-clear-button {
```
Comment (rbarilani) [inline src/styles/molecules/_search-form.scss]: @LeoAref http://stackoverflow.com/questions/19953333/remove-clear-button-on-date-input-ios-safari

Check this apparently there is also something for targeting IE....

Comment (LeoAref) [inline src/styles/molecules/_search-form.scss]: @rbarilani Done

**Comment by gregdeane** [comment]
@LeoAref We had to update the zappr config... would you mind merging `develop` into this PR so we can merge it? Thx!

**Comment by LeoAref** [comment]
@gregdeane Done

**Comment by gabrielhl** [comment]
👍

**Comment by dami-gg** [comment]
:+1:

**Comment by gregdeane** [comment]
👍

**Comment by dami-gg** [comment]
:+1:

**Pull request merged**